2. search/amazon.js

```diff
@@ -7,16 +7,18 @@ const puppeteer = require('puppeteer')
 const screenshot = 'amazon_nyan_cat_pullover.png'
 try {
   (async () => {
-    const browser = await puppeteer.launch()
+    const browser = await puppeteer.launch({ headless: false})
     const page = await browser.newPage()
     await page.setViewport({ width: 1280, height: 800 })
     await page.goto('https://www.amazon.com')
     await page.type('#twotabsearchtextbox', 'nyan cat pullover')
     await page.click('input.nav-input')
-    await page.waitForSelector('#resultsCol')
+    await page.waitFor(3000);
```

Comment:
This is an anti-pattern. `waitFor(3000)` or any wait period that does not wait for an explicit element is bound to break. Please wait for a specific element.